client/src/main/java/com/kritsit/casetracker/shared/domain/model/Permission.java

```diff
@@ -1,8 +1,7 @@
 package com.kritsit.casetracker.shared.domain.model;
 
-import java.io.Serializable;
 
-public enum Permission implements Serializable {
+public enum Permission {
```

Comment:
Serializable was ignored, enums are inherently serializable. I actually learned that enums could exist in their own file through this, thanks :D